docs/technical-design/howto-migrations.md

```diff
@@ -0,0 +1,48 @@
+# How to complete migrations
+
+## Background
+Migrations are a controlled way to change existing records in the database or else change the shape of tables/relationships. They are necessary when fields are added, removed, or when the meaning behind fields and their relationships change.
+
+## General Guidance
+- Prioritize testing in environments that have data similar to production. If you are testing on a review app, you should pre-populate data through using the deployed app a user or via Cypress runs to have a diversity of test data available.
+- Run in all environments, in as close to a way as they run in CI.
```

Comment:
Maybe spell this out a bit more? I'm not sure I know what it means.

Reply:
Oops yes. That is very vague. The long version is I found out that though you could run it from the CLI with `severless` commands which is a lot faster than waiting for PR apps to deploy and merge (which really slows feedback incredibly) @mojotalantikite feels its much better to run through the lambda in AWS because you are running in more of CI-like way in terms of relying on the AWS env. So I think its good to know to prefer that method. I will figure out a couple word way to say that succinctly.